x11-misc/libinput-gestures/libinput-gestures-2.57.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
```

Comment:
Just a small nit pick. Since we already dropped python3.6 support, you can also drop here.